enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -790,7 +790,32 @@ To answer this question, we need to understand the requirements around feature
 gates and understand whether just installing the gatewayclasses CRD may be
 problematic for some clusters.
 
-**Resolution**: TBD.
+**Resolution**: Yes, we should use a feature gate.  It is not just the installation of
+the CRDs that must be considered.  The additional controllers and watchers needed for
```

Comment:
```suggestion
the CRDs that must be considered.  The additional controllers and watches needed for
```
We could delay starting these controllers and watches until the CRDs were created, but this is a valid point.